README.md

```diff
@@ -1,84 +1,90 @@
-# `jest-runner-tsd`
+# jest-runner-tsd
```

Comment:
why?

Reply:
Set in monospace font it was too long for mobile screen ;D Seemed like bold is enough. Not important at all. Can stay.